src/main/resources/db/changesets/createAdminOrganizerUsers-changeset-2019.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd
+    http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+
+    <changeSet id="create roles" author="kostka">
+        <insert tableName="role">
+            <column name="role_id" value="1"/>
+            <column name="role" value="ADMIN"/>
+        </insert>
+        <insert tableName="role">
+            <column name="role_id" value="2"/>
+            <column name="role" value="ORGANIZER"/>
+        </insert>
+        <insert tableName="role">
+            <column name="role_id" value="3"/>
+            <column name="role" value="USER"/>
+        </insert>
+    </changeSet>
+    <changeSet id="create user accounts" author="kostka">
+    <insert tableName="user_account">
+        <column name="user_id" value="1"/>
+        <column name="username" value="admin"/>
+        <!-- the password value is BCrypted string of the "Prazdno" -->
+        <column name="password" value="$2a$10$qs.xj/pIYebwtf2FrchEteOvdQ38qLTuFeOZXm8twXL0uOJFY1kMG"/>
+    </insert>
+    <insert tableName="user_account">
+        <column name="user_id" value="2"/>
+        <column name="username" value="organizer"/>
+        <!-- the password value is BCrypted string of the "4Dprostor" -->
+        <column name="password" value="$2a$10$7s.PnBW19pd8y7dgYrxDCeSflx4P/QI4Mk3TOuoBHLIEt2qQna9wG"/>
```

Comment:
Tak to vypadá, že tady to heslo je pro string "organizer"

Reply:
dobry catch, opraveno :)